content/blog/2021/11/camel-k-release-1-7/index.md

```diff
@@ -0,0 +1,106 @@
+preview: "What's new in Camel K 1.7"
+---
+
+Today we're happy to announce a new major release for Camel K: [Camel K 1.7 version](https://github.com/apache/camel-k/releases/tag/v1.7.0), quite a big leap from latest 1.6 we released a couple of months ago. Keep reading to discover what's new in Camel K! 
+
+First of all, let's highlight the set of technologies on which Camel K 1.7 is based:
+
+* Apache Camel K Runtime 1.10.0
+* Apache Camel Quarkus 2.4.0
+* Apache Camel 3.12.0
+* Apache Camel Kamelets 0.5.0
+
+Thanks to Apache Camel, Camel Quarkus and Kamelet Catalog contributors for the great effor they've put in those new releases as well.
+
+## Dependencies upgrade
+
+The upgrades we applied in this new release, deserve a dedicate section.
+
+We upgraded `Golang` version to `1.6` which gave us the possibility to **bump several Kubernetes API** and so, be in line with the latest enhancements released in K8S space. We also moved to the latest `maven` version, which now point to `3.8.3`. We took the opportunity to promote `Knative` and `Service Binding` to their first `1.x` stable release and to move `Jolokia` to version `1.7.1`. You can find more details on the [GitHub release page](https://github.com/apache/camel-k/releases/tag/v1.7.0).
+
+## Quarkus native build support
+
+The most exciting and promising feature we brought in Camel K 1.7 is probably the support for `Quarkus` native build. If you're not yet aware of it, Quarkus is the great technology that allows your Java application to **run as fast as any native compiled application**. That helps reducing the resource footprint and reduce almost to zero the startup time, making your application a `Cloud Native` first class citizen.
+
+It was a long development and we're still polishing few things before adding official documentation (will do soon, keep tuned). You can find some useful information in the [discussion done during the development](https://github.com/apache/camel-k/pull/2536). The resume is that you can now launch your Camel application with a Quarkus Native profile:
+
+```
+$ kamel run -t quarkus.package-type=native ...
+```
+The build will need some time and some more resource than usual (average test of 5 minutes and 4GB of memory, but it will depend on each case) as there is an ahead of time compilation. The great news is that we thought that, while your native application is building, you can benefit the JVM mode as done normally so far:
+```
+$ kamel run -t quarkus.package-type=fast-jar -t quarkus.package-type=native ...
+```
+The Quarkus JVM application will be replaced with a rolling deployment update as soon as the native build will be completed!
+
+We will provide more official documentation soon. In the while, feel free to try it and report any problem, feedback or suggestion, we'd love to hear about how you're using it.
+
+This feature deserves a special mention to [Antonin Stefanutti](https://github.com/astefanutti) who has lead its development.
```

Comment:
I think that can be removed :) We did it as a team!

Reply:
Nah, it was a great contribution this guy did!! The team is the winner, but let's recognize when someone score a great goal!!